dbuild/templates/dockerfile.jinja

```diff
@@ -1,4 +1,9 @@
 FROM {{ dist }}:{{ release }}
 MAINTAINER {{ maintainer }}
+{%- if http_proxy is defined %}
+RUN echo 'http_proxy="{{ http_proxy }}"' >> /etc/environment
+RUN echo 'https_proxy="{{ https_proxy }}"' >> /etc/environment
```

Comment:
Are these actually required? Doesn't the apt.config on the next line handle everything or are we doing other things in the container that requires a proxy? I'm not sure, to be honest.